src/main/java/org/datadog/jmxfetch/JmxAttribute.java

```diff
@@ -191,6 +194,22 @@ private List<String> getBeanParametersList(
         return beanTags;
     }
 
+    private List<String> normalizeInstanceParameters(List<String> beanParametersList) {
+        List<String> instanceTagsList = new ArrayList<String>(beanParametersList.size());
+        for (String beanParameter : beanParametersList) {
+            if (normalizeBeanParamTags == true) {
+                String oldBeanParamater = beanParameter;
+                beanParameter = beanParameter.replace("\"","");
+                if (!oldBeanParamater.equals(beanParameter)) {
+                    log.info("Removing quotes on bean with parameter: " + oldBeanParamater);
```

Comment:
info is too high a level for this. Maybe `debug` level, but even that may be too high. This will happen a LOT and the logs are not that useful. I'd be inclined to get rid of this log altogether.